dialog/file.go

```diff
@@ -78,17 +82,21 @@ func (f *fileDialog) makeUI() fyne.CanvasObject {
 		if f.file.save {
 			callback := f.file.callback.(func(fyne.URIWriteCloser, error))
 			name := f.fileName.(*widget.Entry).Text
-			path := filepath.Join(f.dir, name)
+			path, _ := storage.Child(f.dir, name)
```

Comment:
Should rename `path` to `location`.

Reply:
Commentary: we agreed that this was the terminology that should be `location` for URIs. I thought I went back and caught the places where I used `path`. I think we could also abbreviate to `loc` if you like.

Reply:
Yeah, in most cases this is right and I'm updating. In one or two it relates to file specific code that we agreed would be removed in another PR